files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/html_and_css/index.md

```diff
@@ -0,0 +1,353 @@
+---
+title: CycleTracker：基础 HTML 和 CSS
```

Comment:
Could you align the translation of [titles](https://developer.mozilla.org/zh-CN/docs/Web/Progressive_web_apps/Tutorials/CycleTracker)

```suggestion
title: 经期跟踪器：基础 HTML 和 CSS
```